TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,95 @@
+fastlane_version "2.44.1"
```

Comment:
Why specify the fastlane version? Seems like this will need to be updated a _lot_ in the template.